packages/strings/src/normalize.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Map of characters to their base characters
+ */
+const CHAR_MAP = {
+	À: 'A',
+	Á: 'A',
+	Â: 'A',
+	Ã: 'A',
+	Ä: 'A',
+	Å: 'A',
+	Æ: 'A',
+	Ç: 'C',
+	È: 'E',
+	É: 'E',
+	Ê: 'E',
+	Ë: 'E',
+	Ì: 'I',
+	Í: 'I',
+	Î: 'I',
+	Ï: 'I',
+	Ð: 'D',
+	Ñ: 'N',
+	Ò: 'O',
+	Ó: 'O',
+	Ô: 'O',
+	Õ: 'O',
+	Ö: 'O',
+	Ø: 'O',
+	Ù: 'U',
+	Ú: 'U',
+	Û: 'U',
+	Ü: 'U',
+	Ý: 'Y',
+	Þ: 'B',
+	ß: 'Ss',
+	à: 'a',
+	á: 'a',
+	â: 'a',
+	ã: 'a',
+	ä: 'a',
+	å: 'a',
+	æ: 'a',
+	ç: 'c',
+	è: 'e',
+	é: 'e',
+	ê: 'e',
+	ë: 'e',
+	ì: 'i',
+	í: 'i',
+	î: 'i',
+	ï: 'i',
+	ð: 'd',
+	ñ: 'n',
+	ò: 'o',
+	ó: 'o',
+	ô: 'o',
+	õ: 'o',
+	ö: 'o',
+	ø: 'o',
+	ù: 'u',
+	ú: 'u',
+	û: 'u',
+	ü: 'u',
+	ý: 'y',
+	þ: 'b',
+	ÿ: 'y'
+} as const;
+
+
+/**
+ * Replaces all accents with their base characters in a case sensitive manner
+ */
+export const normalize = (text: string) => {
```

Comment:
I had this version in a project, I wonder if there are any differences:

```ts
export const unaccent = (str: Maybe<string>) => {
	if (!str) {
		return '';
	}
	return decodeURIComponent(str)
		.normalize('NFD')
		.replace(/\p{Diacritic}/gu, '');
};
```

Reply:
It might get rid of emojis if I understand correctly. Plus requires some api FF might now have. Plus NFD form create bugs in my tests when I tried it..